addons/stock/wizard/stock_backorder_confirmation.py

```diff
@@ -10,7 +10,7 @@ class StockBackorderConfirmationLine(models.TransientModel):
     _description = 'Backorder Confirmation Line'
 
     backorder_confirmation_id = fields.Many2one('stock.backorder.confirmation', 'Immediate Transfer')
-    picking_id = fields.Many2one('stock.picking', 'Transfer')
+    picking_id = fields.Many2one('stock.picking', 'Transfer', domain="[('id', '=', context.get('active_id'))]")
```

Comment:
Should be `readonly` as discussed on the project task.

Reply:
Hello @Whenrow ,

We cannot set picking_id to readonly because if we do, we won't be able to get the value of picking_id inside the process() function so this function creates a backorder for each picking, regardless of whether the to_backorder value is true or false.

Thank you!

Reply:
This is because the pickings are given to the wizard via `default_pick_ids` in the context then dispatched on newly created confirmation lines. As they are `readonly=1`, the value given is not save. (this is the default behaviour of readonly field. To force the save of those values. You need to add `force_save="1"` on the `<field name="picking_id">` element of the view

Reply:
Hello @Whenrow ,

Changes have been made. And thanks for the suggestion.